main/parse.c

```diff
@@ -1635,7 +1635,7 @@ static boolean enableLanguageKind (
 }
 
 static boolean enableLanguageKindLong (
-	const langType language, const const char *kindLong, const boolean mode)
+	const langType language, const char *kindLong, const boolean mode)
```

Comment:
seeing the other arguments, I would think it was supposed to be `const char *const kindLong`